### DISCUSSION TITLE
MobilePay support

### DISCUSSION THREAD

**Discussion by ce2kettu** [flutter-stripe/flutter_stripe#1625] [Q&A]
Hey! MobilePay is currently in [invite-only beta](https://stripe.com/docs/payments/mobilepay). Is it possible to add its payment method bindings to this library? Or is there another workaround?

**Comment by viskanic** [comment]
Hello 👋 

We are currently developing a Flutter app and the client requested support for [MobilePay](https://www.mobilepay.dk/).
Are there any plans to support MobilePay at some point in the future?

Any information would be much appreciated.
We are looking forward to your reply.

BR

**Reply by Shahidbangash** [reply]
i am currently working on creating a package that supports mobile pay using Stripe Sdk .

i will post update for you when it is ready

**Comment by remonh87** [comment]
Mobile pay should just work with this sdk. You just need to handle the deeplink coming back https://docs.stripe.com/payments/mobilepay/accept-a-payment?web-or-mobile=mobile&payments-ui-type=react-native .
